lib/MusicBrainz/Server/Controller/User.pm

```diff
@@ -655,11 +657,15 @@ sub tag : Chained('load_tag') PathPart('')
     if ($tag) {
         %tagged_entities = map {
             my $entity_properties = $ENTITIES{$_};
+            my $model = $c->model($entity_properties->{model});
 
-            my ($entities, $total) = $c->model($entity_properties->{model})->tags->find_editor_entities(
+            my ($entities, $total) = $model->tags->find_editor_entities(
                 $user->id, $tag->id, $show_downvoted, 10, 0);
-            $c->model('ArtistCredit')->load(map { $_->entity } @$entities)
+            my @entity_entries = map { $_->entity } @$entities;
+            $c->model('ArtistCredit')->load(@entity_entries)
                 if $entity_properties->{artist_credits};
+            $model->load_aliases(@entity_entries)
+                if $model->can('load_aliases');
```

Comment:
This should work, right? The intent seems a bit clearer, since entities.json is supposed to be the source of truth. That is, if it's supposed to support aliases but doesn't have a `load_aliases` method, it should probably ISE.
```suggestion
                if $entity_properties->{aliases};
```

Reply:
It should but we use `->can` everywhere else. As such, we should probably either keep using it, or change all of them in a separate commit?

Reply:
I added a separate commit changing *some* of these (all the ones where we already load and use `entity_properties` because of ACs). If you want we can change all of them, it'll just involve loading `%ENTITIES` in more places.